src/App.js

```diff
@@ -1,26 +1,28 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+
+  const addmovies = (newMovie) => {
+    if (movies.find(({ imdbId }) => imdbId === newMovie.imdbId)) {
+      return;
+    }
 
-  render() {
-    const { movies } = this.state;
+    setMovies(() => [...movies, newMovie]);
+  };
 
-    return (
-      <div className="page">
-        <div className="page-content">
-          <MoviesList movies={movies} />
-        </div>
-        <div className="sidebar">
-          <FindMovie />
-        </div>
+  return (
+    <div className="page">
+      <div className="page-content">
+        <MoviesList movies={movies} />
+      </div>
+      <div className="sidebar">
+        <FindMovie onMovies={addmovies} />
```

Comment:
```suggestion
        <FindMovie onAdd={addmovies} />
```
Not really clear what should `onMovies` do